src/validation/EditUserSchema.js

```diff
@@ -0,0 +1,7 @@
+import * as Yup from 'yup';
+
+export default Yup.object().shape({
+  role: Yup.string()
+    .oneOf(['ADMIN', 'TEACHER', 'STUDENT'])
+    .required('Role is required'),
+});
```

Comment:
Hey, just one small change with the schema. Not this schema, but the `src/validation/NewUserSchema.js`, needs `useremail` on line 4 to be just `email`. Sorry, I figured this would be the best place to put this. At least the files are related.

Reply:
Fixed.